src/components/ui/StartButton.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div
+    class="wrap"
+    v-if="$store.state.theme"
+    :class="{ [$store.state.theme.buttonStyle]: true }"
+  >
+    <button class="main_button" @click="hanldeClick" @mouseenter="hanldeHover">
+      {{ text }}
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "StartButton",
+  props: ["text"],
+  data: function () {
+    return {};
+  },
+  mounted() {},
+  methods: {
+    hanldeHover() {
+      this.$store.state.audio.playHoverEffect("ui/click2");
+    },
+    hanldeClick() {
+      this.$store.state.audio.playEffect("ui/power");
+      this.$emit("click", null);
```

Comment:
What does this do? Calling a method with a `null` parameter seems quite strange lol.

Reply:
Good question, I think I can just omit the second param